adhocracy4/exports/views.py

```diff
@@ -48,32 +47,33 @@ def _clean_field(self, field):
         return field
 
 
-class SimpleItemExportView(AbstractXlsxExportView,
-                           VirtualFieldMixin):
+class BaseExport(VirtualFieldMixin):
```

Comment:
after relying on the not implemented `get_object_list()` method this should probably be called `AbstractBaseExport` or `AbstractObjectListExport`